Docs/beta/conf.py

```diff
@@ -181,14 +181,21 @@ def setup(app):
 html_sidebars = {
     "**": [
         "sidebar/brand.html",
+        "sidebar/doc_versions.html",
+        "sidebar/search.html",
         "sidebar/scroll-start.html",
         "sidebar/navigation.html",
-#        "sidebar/search.html",
         "sidebar/scroll-end.html",
     ]
 }
 # Custom TOC settings
 
+# Parameters for use in templates
+
+html_context = {
+  'current_version' : "1.36",
+  'versions_page' : '<a href="https://quic.github.io/aimet-pages/releases/latest/versions.html">Other versions</a>'
```

Comment:
Seems like this link doesn't exist today
Is this a placeholder that you will fill out?

Reply:
@quic-kyunggeu Yes, this is a placeholder. Since there isn't built-in support in Sphinx for maintaining a version history, I think we need to discuss how we're going to store and serve previous versions of the site.